File: Medium/Python/1395. Count Number of Teams.py
class Solution(object):
    def numTeams(self, rating):
        count = 0

        for i in range(len(rating)):
            left = 0
            right = 0
            for j in range(0, i):
                if rating[j] < rating[i]:
                    left += 1
            for k in range(i + 1, len(rating)):
                if rating[i] < rating[k]:
                    right += 1
            count += (left * right) + (i - left) * (len(rating) - i - right - 1)

        return count

File: Medium/JavaScript/1395. Count Number of Teams.js
var numTeams = function(rating) {
    let count = 0;

    for(let i = 0; i < rating.length; i++){
        let left = 0;
        let right = 0;
        for(let j = 0; j < i; j++){
            if(rating[j] < rating[i])
                left++;
        }
        for(let k = i + 1; k < rating.length; k++){
            if(rating[i] < rating[k])
                right++;
        }
        count += (left * right) + (i - left) * (rating.length - i - right - 1);
    }

    return count;
};
